tests/attr/test_multiscale_fastcam.py

```diff
@@ -0,0 +1,2024 @@
+#!/usr/bin/env python3
+import unittest
+from typing import Any, Dict
+
+import torch
+
+from captum.attr._core.multiscale_fast_cam import MultiscaleFastCam
+
+from ..helpers.basic import BaseTest, assertTensorAlmostEqual
+from ..helpers.basic_models import BasicModel_ConvNet
+
```

Comment:
It would also be good to add DataParallel tests and ensure all functionality works appropriately on GPU. You can take a look at the test generator here: https://github.com/pytorch/captum/blob/master/tests/attr/test_data_parallel.py, which is based on this config https://github.com/pytorch/captum/blob/master/tests/attr/helpers/test_config.py.